__tests__/cars.test.js

```diff
@@ -0,0 +1,93 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('Car Routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('/cars/:id should return a single bird', async () => {
```

Comment:
you want your test description to match the resource you're testing for so it doesn't look like you just copy-pasted everything